settings.py

```diff
@@ -55,6 +58,13 @@
     'submissions'
 )
 
+# TODO: These are removed from global defaults. Not sure we need here or not but i have added this to remove warning.
```

Comment:
Not sure about this: https://docs.djangoproject.com/en/1.8/ref/settings/#middleware-classes